src/mongocxx/options/bulk_write.hpp

```diff
@@ -18,6 +18,8 @@
 
 #include <bsoncxx/stdx/optional.hpp>
 
+#include <mongocxx/write_concern.hpp>
```

Comment:
Unrelated, but I needed to move this over here form the cpp file in order to compile. We do use the write concern in our hpp file, so unless we want to forward declare the class, I think we need it here.

Currently investigating why this isn't an issue in other branches...